back/config/environments/development.rb

```diff
@@ -44,7 +44,7 @@
   config.action_mailer.perform_caching = false
 
   # Previewing mails from the email engine.
-  config.action_mailer.preview_path = "#{Rails.root}/engines/free/email_campaigns/spec/mailers/previews"
+  config.action_mailer.preview_path = "#{Rails.root}/engines/free/user_confirmation/spec/mailers/previews"
```

Comment:
That's annoying 😅 I think for sure, we should make all previews available, so for me this would need to change back to `email_campaigns`. It seems acceptable for me to just put the email campaign in the email campaigns engine if there's no easy alternative. Shouldn't we reserve some development time on continuations on the open source work? It seems important to me to find a good solution here, as this situation is likely to come back in future epics. 

Perhaps a relatively easy solution could be to have all open source emails in the email campaigns engine and all enterprise email files could get copied by the enable EE script?